modules/standalone-georepl-site-removal.adoc

```diff
@@ -0,0 +1,74 @@
+:_content-type: PROCEDURE
+[id="standalone-georepl-site-removal"]
+= Removing a geo-replicated site from your standalone {productname} deployment
+
+By using the following procedure, {productname} administrators can remove sites in a geo-replicated setup. 
+
+.Prerequisites 
+
+* You have configured {productname} geo-replication with at least two sites, for example, `usstorage` and `eustorage`. 
+* Each site has its own Organization, Repository, and image tags. 
```

Comment:
This description is not correct, actually only the imagestorageplacement is associated with storagelocation.

Reply:
@LiZhang19817 there are three tables that govern blob storage in Quay:

1) `imagestorage`: contains references to all blobs stored in the registry.
2) `imagestoragelocation`: contains the names of all defined storage engines.
3) `imagestorageplacement`: a relational table that connects `imagestorage` with `imagestoragelocation`, i.e. every blob stored in the registry with a particular defined storage engine.

I'm not sure what you mean with your comment and how is it relevant to what you quoted here. Can you please explain what you meant?

Reply:
@ibazulic Quay Geo-Replication means multiple Quay deployments across different sites, each organization created not belonging to any site, why here says **"Each site has its own Organization"**, for example when login from Quay site1 with users 'user1', it's not reasonable to say site1 own this new org.

Reply:
@LiZhang19817 I know what georeplication means and how it works :-) Yep, now that I read it again, I understand better what you meant. Thanks for the additional explanation!